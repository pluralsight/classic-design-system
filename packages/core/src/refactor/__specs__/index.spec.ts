import { cssVars } from '..'

test('tokens', () => {
  expect(cssVars).toMatchSnapshot()
})
