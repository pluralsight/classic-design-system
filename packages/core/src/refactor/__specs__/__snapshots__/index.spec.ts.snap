// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tokens 1`] = `
Object {
  "breakpoints": Object {
    "large": "var(--ps-breakpoints-large)",
    "medium": "var(--ps-breakpoints-medium)",
    "small": "var(--ps-breakpoints-small)",
    "xLarge": "var(--ps-breakpoints-xlarge)",
    "xSmall": "var(--ps-breakpoints-xsmall)",
  },
  "colorsBackgroundDark": Object {
    "1": "var(--ps-colors-background-dark-1)",
    "2": "var(--ps-colors-background-dark-2)",
    "3": "var(--ps-colors-background-dark-3)",
  },
  "colorsBackgroundLight": Object {
    "1": "var(--ps-colors-background-light-1)",
    "2": "var(--ps-colors-background-light-2)",
    "3": "var(--ps-colors-background-light-3)",
  },
  "colorsBackgroundUtility": Object {
    "10": "var(--ps-colors-background-utility-10)",
    "15": "var(--ps-colors-background-utility-15)",
    "20": "var(--ps-colors-background-utility-20)",
    "25": "var(--ps-colors-background-utility-25)",
    "30": "var(--ps-colors-background-utility-30)",
    "40": "var(--ps-colors-background-utility-40)",
    "60": "var(--ps-colors-background-utility-60)",
  },
  "colorsBackgroundUtilityCsv": Object {
    "colorsBackgroundUtilityCsv": "var(--ps-colors-background-utility-csv)",
  },
  "colorsBlack": Object {
    "colorsBlack": "var(--ps-colors-black)",
  },
  "colorsBlue": Object {
    "1": "var(--ps-colors-blue-1)",
    "10": "var(--ps-colors-blue-10)",
    "2": "var(--ps-colors-blue-2)",
    "3": "var(--ps-colors-blue-3)",
    "4": "var(--ps-colors-blue-4)",
    "5": "var(--ps-colors-blue-5)",
    "6": "var(--ps-colors-blue-6)",
    "7": "var(--ps-colors-blue-7)",
    "8": "var(--ps-colors-blue-8)",
    "9": "var(--ps-colors-blue-9)",
  },
  "colorsBorder": Object {
    "highOnDark": "var(--ps-colors-border-high-on-dark)",
    "highOnLight": "var(--ps-colors-border-high-on-light)",
    "lowOnDark": "var(--ps-colors-border-low-on-dark)",
    "lowOnLight": "var(--ps-colors-border-low-on-light)",
  },
  "colorsCode": Object {
    "blue": "var(--ps-colors-code-blue)",
    "gray": "var(--ps-colors-code-gray)",
    "green": "var(--ps-colors-code-green)",
    "orange": "var(--ps-colors-code-orange)",
    "purple": "var(--ps-colors-code-purple)",
    "red": "var(--ps-colors-code-red)",
    "sand": "var(--ps-colors-code-sand)",
    "turquoise": "var(--ps-colors-code-turquoise)",
    "white": "var(--ps-colors-code-white)",
    "yellow": "var(--ps-colors-code-yellow)",
  },
  "colorsGradient": Object {
    "flowBackground": "var(--ps-colors-gradient-flow-background)",
    "flowStop0": "var(--ps-colors-gradient-flow-stop-0)",
    "flowStop100": "var(--ps-colors-gradient-flow-stop-100)",
    "skillsBackground": "var(--ps-colors-gradient-skills-background)",
    "skillsStop0": "var(--ps-colors-gradient-skills-stop-0)",
    "skillsStop100": "var(--ps-colors-gradient-skills-stop-100)",
  },
  "colorsGreen": Object {
    "1": "var(--ps-colors-green-1)",
    "10": "var(--ps-colors-green-10)",
    "2": "var(--ps-colors-green-2)",
    "3": "var(--ps-colors-green-3)",
    "4": "var(--ps-colors-green-4)",
    "5": "var(--ps-colors-green-5)",
    "6": "var(--ps-colors-green-6)",
    "7": "var(--ps-colors-green-7)",
    "8": "var(--ps-colors-green-8)",
    "9": "var(--ps-colors-green-9)",
  },
  "colorsLime": Object {
    "1": "var(--ps-colors-lime-1)",
    "10": "var(--ps-colors-lime-10)",
    "2": "var(--ps-colors-lime-2)",
    "3": "var(--ps-colors-lime-3)",
    "4": "var(--ps-colors-lime-4)",
    "5": "var(--ps-colors-lime-5)",
    "6": "var(--ps-colors-lime-6)",
    "7": "var(--ps-colors-lime-7)",
    "8": "var(--ps-colors-lime-8)",
    "9": "var(--ps-colors-lime-9)",
  },
  "colorsOrange": Object {
    "1": "var(--ps-colors-orange-1)",
    "10": "var(--ps-colors-orange-10)",
    "2": "var(--ps-colors-orange-2)",
    "3": "var(--ps-colors-orange-3)",
    "4": "var(--ps-colors-orange-4)",
    "5": "var(--ps-colors-orange-5)",
    "6": "var(--ps-colors-orange-6)",
    "7": "var(--ps-colors-orange-7)",
    "8": "var(--ps-colors-orange-8)",
    "9": "var(--ps-colors-orange-9)",
  },
  "colorsPink": Object {
    "1": "var(--ps-colors-pink-1)",
    "10": "var(--ps-colors-pink-10)",
    "2": "var(--ps-colors-pink-2)",
    "3": "var(--ps-colors-pink-3)",
    "4": "var(--ps-colors-pink-4)",
    "5": "var(--ps-colors-pink-5)",
    "6": "var(--ps-colors-pink-6)",
    "7": "var(--ps-colors-pink-7)",
    "8": "var(--ps-colors-pink-8)",
    "9": "var(--ps-colors-pink-9)",
  },
  "colorsPrimaryAction": Object {
    "background": "var(--ps-colors-primary-action-background)",
    "textDarkTheme": "var(--ps-colors-primary-action-text-dark-theme)",
    "textLightTheme": "var(--ps-colors-primary-action-text-light-theme)",
  },
  "colorsPurple": Object {
    "1": "var(--ps-colors-purple-1)",
    "10": "var(--ps-colors-purple-10)",
    "2": "var(--ps-colors-purple-2)",
    "3": "var(--ps-colors-purple-3)",
    "4": "var(--ps-colors-purple-4)",
    "5": "var(--ps-colors-purple-5)",
    "6": "var(--ps-colors-purple-6)",
    "7": "var(--ps-colors-purple-7)",
    "8": "var(--ps-colors-purple-8)",
    "9": "var(--ps-colors-purple-9)",
  },
  "colorsRed": Object {
    "1": "var(--ps-colors-red-1)",
    "10": "var(--ps-colors-red-10)",
    "2": "var(--ps-colors-red-2)",
    "3": "var(--ps-colors-red-3)",
    "4": "var(--ps-colors-red-4)",
    "5": "var(--ps-colors-red-5)",
    "6": "var(--ps-colors-red-6)",
    "7": "var(--ps-colors-red-7)",
    "8": "var(--ps-colors-red-8)",
    "9": "var(--ps-colors-red-9)",
  },
  "colorsStatus": Object {
    "error": "var(--ps-colors-status-error)",
    "info": "var(--ps-colors-status-info)",
    "success": "var(--ps-colors-status-success)",
    "warning": "var(--ps-colors-status-warning)",
  },
  "colorsTeal": Object {
    "1": "var(--ps-colors-teal-1)",
    "10": "var(--ps-colors-teal-10)",
    "2": "var(--ps-colors-teal-2)",
    "3": "var(--ps-colors-teal-3)",
    "4": "var(--ps-colors-teal-4)",
    "5": "var(--ps-colors-teal-5)",
    "6": "var(--ps-colors-teal-6)",
    "7": "var(--ps-colors-teal-7)",
    "8": "var(--ps-colors-teal-8)",
    "9": "var(--ps-colors-teal-9)",
  },
  "colorsTextIcon": Object {
    "highOnDark": "var(--ps-colors-text-icon-high-on-dark)",
    "highOnLight": "var(--ps-colors-text-icon-high-on-light)",
    "lowOnDark": "var(--ps-colors-text-icon-low-on-dark)",
    "lowOnLight": "var(--ps-colors-text-icon-low-on-light)",
  },
  "colorsWhite": Object {
    "colorsWhite": "var(--ps-colors-white)",
  },
  "colorsYellow": Object {
    "1": "var(--ps-colors-yellow-1)",
    "10": "var(--ps-colors-yellow-10)",
    "2": "var(--ps-colors-yellow-2)",
    "3": "var(--ps-colors-yellow-3)",
    "4": "var(--ps-colors-yellow-4)",
    "5": "var(--ps-colors-yellow-5)",
    "6": "var(--ps-colors-yellow-6)",
    "7": "var(--ps-colors-yellow-7)",
    "8": "var(--ps-colors-yellow-8)",
    "9": "var(--ps-colors-yellow-9)",
  },
  "layers": Object {
    "globalDropdown": "var(--ps-layers-global-dropdown)",
    "globalSidenav": "var(--ps-layers-global-sidenav)",
    "globalSkipToContentBanner": "var(--ps-layers-global-skip-to-content-banner)",
    "globalTopnav": "var(--ps-layers-global-topnav)",
    "main": "var(--ps-layers-main)",
  },
  "layout": Object {
    "spacingLarge": "var(--ps-layout-spacing-large)",
    "spacingMedium": "var(--ps-layout-spacing-medium)",
    "spacingSmall": "var(--ps-layout-spacing-small)",
    "spacingXLarge": "var(--ps-layout-spacing-xlarge)",
    "spacingXSmall": "var(--ps-layout-spacing-xsmall)",
    "spacingXXLarge": "var(--ps-layout-spacing-xxlarge)",
    "spacingXXSmall": "var(--ps-layout-spacing-xxsmall)",
  },
  "motion": Object {
    "speedFast": "var(--ps-motion-speed-fast)",
    "speedNormal": "var(--ps-motion-speed-normal)",
    "speedSlow": "var(--ps-motion-speed-slow)",
    "speedXFast": "var(--ps-motion-speed-xfast)",
    "speedXSlow": "var(--ps-motion-speed-xslow)",
  },
  "type": Object {
    "fontFamily": "var(--ps-type-font-family)",
    "fontFamilyCode": "var(--ps-type-font-family-code)",
    "fontSize100": "var(--ps-type-font-size-100)",
    "fontSize1000": "var(--ps-type-font-size-1000)",
    "fontSize1100": "var(--ps-type-font-size-1100)",
    "fontSize1200": "var(--ps-type-font-size-1200)",
    "fontSize200": "var(--ps-type-font-size-200)",
    "fontSize300": "var(--ps-type-font-size-300)",
    "fontSize400": "var(--ps-type-font-size-400)",
    "fontSize500": "var(--ps-type-font-size-500)",
    "fontSize600": "var(--ps-type-font-size-600)",
    "fontSize700": "var(--ps-type-font-size-700)",
    "fontSize800": "var(--ps-type-font-size-800)",
    "fontSize900": "var(--ps-type-font-size-900)",
    "fontSizeBase": "var(--ps-type-font-size-base)",
    "fontSizeGigantic": "var(--ps-type-font-size-gigantic)",
    "fontSizeJumbo": "var(--ps-type-font-size-jumbo)",
    "fontSizeLarge": "var(--ps-type-font-size-large)",
    "fontSizeMedium": "var(--ps-type-font-size-medium)",
    "fontSizeSmall": "var(--ps-type-font-size-small)",
    "fontSizeXLarge": "var(--ps-type-font-size-xlarge)",
    "fontSizeXSmall": "var(--ps-type-font-size-xsmall)",
    "fontSizeXXLarge": "var(--ps-type-font-size-xxlarge)",
    "fontWeight100": "var(--ps-type-font-weight-100)",
    "fontWeight200": "var(--ps-type-font-weight-200)",
    "fontWeight300": "var(--ps-type-font-weight-300)",
    "fontWeight400": "var(--ps-type-font-weight-400)",
    "fontWeight500": "var(--ps-type-font-weight-500)",
    "fontWeight600": "var(--ps-type-font-weight-600)",
    "fontWeight700": "var(--ps-type-font-weight-700)",
    "fontWeight800": "var(--ps-type-font-weight-800)",
    "fontWeight900": "var(--ps-type-font-weight-900)",
    "fontWeightBlack": "var(--ps-type-font-weight-black)",
    "fontWeightBold": "var(--ps-type-font-weight-bold)",
    "fontWeightBook": "var(--ps-type-font-weight-book)",
    "fontWeightDefault": "var(--ps-type-font-weight-default)",
    "fontWeightDemiBold": "var(--ps-type-font-weight-demi-bold)",
    "fontWeightLight": "var(--ps-type-font-weight-light)",
    "fontWeightMedium": "var(--ps-type-font-weight-medium)",
    "fontWeightRegular": "var(--ps-type-font-weight-regular)",
    "fontWeightStrong": "var(--ps-type-font-weight-strong)",
    "fontWeightThin": "var(--ps-type-font-weight-thin)",
    "fontWeightXBold": "var(--ps-type-font-weight-xbold)",
    "fontWeightXLight": "var(--ps-type-font-weight-xlight)",
    "letterSpacingAllCaps": "var(--ps-type-letter-spacing-all-caps)",
    "letterSpacingGigantic": "var(--ps-type-letter-spacing-gigantic)",
    "letterSpacingJumbo": "var(--ps-type-letter-spacing-jumbo)",
    "letterSpacingLarge": "var(--ps-type-letter-spacing-large)",
    "letterSpacingLoose": "var(--ps-type-letter-spacing-loose)",
    "letterSpacingLooser": "var(--ps-type-letter-spacing-looser)",
    "letterSpacingMedium": "var(--ps-type-letter-spacing-medium)",
    "letterSpacingNone": "var(--ps-type-letter-spacing-none)",
    "letterSpacingSmall": "var(--ps-type-letter-spacing-small)",
    "letterSpacingTight": "var(--ps-type-letter-spacing-tight)",
    "letterSpacingTighter": "var(--ps-type-letter-spacing-tighter)",
    "letterSpacingXLarge": "var(--ps-type-letter-spacing-xlarge)",
    "letterSpacingXSmall": "var(--ps-type-letter-spacing-xsmall)",
    "letterSpacingXXLarge": "var(--ps-type-letter-spacing-xxlarge)",
    "lineHeightExtra": "var(--ps-type-line-height-extra)",
    "lineHeightHuge": "var(--ps-type-line-height-huge)",
    "lineHeightStandard": "var(--ps-type-line-height-standard)",
    "lineHeightTight": "var(--ps-type-line-height-tight)",
  },
}
`;
